### PR TITLE
(Perks mod) Create the trouble seeker perk

### PR DESCRIPTION
**Pull request by SariusSkelrets**
#### Summary
Mods "Adds a new perk to the perk mod"

#### Purpose of change

Add a perk that will regularly spawn a zombie in the reality bubble when activated.

Reopens #64760 and update its files to include the new perks

#### Describe the solution

Add that perk that spawns a zombie every ~4-7 hours in the reality bubble at a distance of 35-50 tiles from the player, for those who want more zombies to kill, loot and butcher.

#### Describe alternatives you've considered

Creating another perk

#### Testing

Got the perk, activated it, was attacked by a zombie

Tried to toggle the perk, activated and deactivated as expected

#### Additional context

This Pr is ready to be merged

### CODE REVIEW

**Comment by SariusSkelrets** [comment]
Got a little busy with work and wasn’t able to find time to apply the fix

I’ll soon update the PR and apply the recycler fix

**Comment by SariusSkelrets** [comment]
Tested it. It works as intended now

**Comment by bombasticSlacks** [comment]
Hey you need to lint your files for this to run successfuly https://dev.narc.ro/cataclysm/format.html

**Comment by SariusSkelrets** [comment]
Trying to fix some merge conflicts right now

**Comment by SariusSkelrets** [comment]
The conflicts are fixed now

**Comment by bombasticSlacks** [comment]
I'm going to push a fix to your branch 😄

**Review by bombasticSlacks — APPROVED** [review]

File: data/mods/BombasticPerks/perks.json [hunk 1]
```diff
@@ -238,6 +238,20 @@
     "processed_eocs": [ "EOC_can_hunting" ],
     "time": 24000
   },
+  {
+    "type": "mutation",
+    "id": "perk_troubleseeker",
+    "name": { "str": "Trouble seeker" },
+    "points": 0,
+    "description": "As if the apocalypse itself wasn't enough, you are constantly looking for some trouble.  Zombies will sometimes appear out of nowhere to attack you.  Fortunately, they always appear at a good distance from you. Activate to toggle.",
```
Comment (bombasticSlacks) [inline data/mods/BombasticPerks/perks.json:246]:
```suggestion
    "description": "As if the apocalypse itself wasn't enough, you are constantly looking for some trouble.  Zombies will sometimes appear out of nowhere to attack you.  Fortunately, they always appear at a good distance from you.  Activate to toggle.",
```

**Comment by bombasticSlacks** [comment]
Thanks for doing this, I was really slacking

If you do another perk I've rewritten the style to be much easier to add, not as much repetitive code.

**Pull request merged**